psydac/api/ast/evaluation.py

```diff
@@ -22,557 +18,10 @@
 
 from .basic     import SplBasic
 from .utilities import build_pythran_types_header, variables
-from .utilities import filter_loops, filter_product, select_loops
 from .utilities import rationalize_eval_mapping
 from .utilities import compute_atoms_expr_mapping
 from .utilities import compute_atoms_expr_field
 
-#==============================================================================
-class EvalQuadratureMapping(SplBasic):
```

Comment:
can you keep this, we have a glt test which is marked `xfail`, it needs to be fixed

Reply:
We are not using this code anywhere in Psydac.
Are you sure that we need all of this just for the GLT expressions?
Can't we use the same machinery that we use everywhere else?

Reply:
the glt uses the old mechanism, what I implemented works only for fem

Reply:
I see. Do we need really a separate process for GLT expressions? What you have done for FEM cannot be reused to some extent?

Reply:
it can, but it needs time